### DISCUSSION TITLE
[Q-Select] Dropdown is forced to top with long list

### DISCUSSION THREAD

**Discussion by jimmiejackson414** [quasarframework/quasar#12593] [General - Components / Directives / etc] [answered]
I've got a basic form for a user to fill in details about themselves such as first/last name, email and address. The last field is for country, in which I'm using the q-select component populated with an array of countries (all ~239 of them). When the user clicks, the dropdown menu is forced above the input and underneath my navbar however, and I'd like to be able to force it below the input.

I understand that this is automatically calculated based on whether it "fits" below first, if my understanding of [this discussion](https://github.com/quasarframework/quasar/discussions/new) is correct, so I believe that in order for me to fix this I would simply need to add some sort of max-height to the dropdown? I've pored over the docs but can't seem to find a way to set a prop or anything for the max-height of the dropdown. Is this possible some way?

**Comment by MilosPaunovic** [comment, marked as answer]
You can use the `popup-content-class` prop of `QSelect` to do it:
[https://codepen.io/milospaunovic/pen/rNYZoww](https://codepen.io/milospaunovic/pen/rNYZoww)
